### ISSUE TITLE
Minified build in npm package makes auditing harder

### ISSUE THREAD

**Issue by joepie91** [dchest/tweetnacl-js#204]
Hi,

TweetNaCl.js currently includes a minified build in its package on npm, but unfortunately this is making dependency auditing quite a bit harder; now in addition to a human-readable version, a minified version now also needs to be audited and/or reproduced (which has its own toolchain trust issues).

I've written a bit more about this topic (and why minified builds are not useful on npm) [here](https://gist.github.com/joepie91/04cc8329df231ea3e262dffe3d41f848) -- I'd like to request removing it from the npm package :)

**Comment by dchest** [comment]
Makes sense. Note that the [default import uses non-minefield version](https://github.com/dchest/tweetnacl-js/blob/f1ec050ceae0861f34280e62498b1d3ed9c350c6/package.json#L5), so unless the user of the library imports a minified file explicitly, nacl-fast.js will be used.

I’ve marked this for 2.0 version, since removing minified builds would be a breaking change.

Thanks!

**Comment by dchest** [comment]
*no-minified. But I like that autocorrect turned it into “non-minefield” 😄

**Comment by joepie91** [comment]
Great, thanks :)
